### ISSUE TITLE
18789704: VoiceOver reads "1m" as "One meter" no matter the context

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#2838]
#### Description

Summary:
VoiceOver always assumes that a number followed with "m" is a value in meters and reads it as such by saying "{number} meter(s)". This is particularly problematic for apps that have fuzzy/relative dates (e.g., "1m ago").

This can be reproduced in English and Spanish, and potentially any other language in which "m" is used as a shorthand for meters.

VoiceOver should identify these cases as dates. Alternatively, there should be a way to opt out of this behavior or annotate the text (perhaps with a NSAttributedString attribute?) so that VoiceOver does not interpret "m" as "meters".

Steps to Reproduce:
Set the device language to English
Create a sample app that has a label with text "1m ago".
Open the app.
Turn VoiceOver on.
Select the label.

Expected Results:
VoiceOver says "One em ago".

Actual Results:
VoiceOver says "One meter ago".
## 

Product Version: 8.0.2 (12A405)
Created: 2014-10-28T00:12:18.179080
Originated: 2014-10-27T00:00:00
Open Radar Link: http://www.openradar.me/18789704
